kunstmaan/bundles-cms/4.0/etc/packages/kunstmaan_liip_imagine.yaml

```diff
@@ -0,0 +1,75 @@
+liip_imagine:
```

Comment:
This recipe defines too many things. It should not configure external libs/bundles like this one.